### ISSUE TITLE
*Request* Feature develop request

### ISSUE THREAD

**Issue by LukasW1337** [Thiry1/csgo-custom-hud#24]
I know you are proberbly doing this in your freetime - @Thiry1 - But i have a request for you 😄 

We have a "big" tournament starting the 21. of May, and i only need 2 features and 1 bug fixed before everything is so perfect!

Overtime mode - Switch the text 

> "Round x / 30"
to 
> "OVERTIME Round x / 6"

Then a BO3 Feature of some kind would be really helpful, i know both of these things are on the roadmap, but a manual way of doing them for now would be nice.

Then the bug i reported in another issue is critical 😆 

Love the overlay - love the work you are doing, keep it up!

**Comment by Thiry1** [comment]
I will do.
But I may not make it in time for that day.

**Comment by bergmannf** [comment]
I started working on the seconds feature (BO3 / BO5 markers), but there seem to be some problems when trying to run it against official demos:

It seems many servers do *not* correctly set the `matches_won_this_series` and `num_matches_to_win_series` in the game state messages.

As an ugly workaround I read the scores from a `teamScores.json` file instead that is reread every round-event - for anyone wanting to give it a try you can check it out in my [fork](https://github.com/bergmannf/csgo-custom-hud/tree/bo_markers).

I didn't want to open a pull request though, as it is terribly hacky with the scores-file, but I guess some kind of alternative way to specify the scores is needed, as *no* match I checked had that correct numbers set.

**Comment by Thiry1** [comment]
>It seems many servers do not correctly set the matches_won_this_series and num_matches_to_win_series in the game state messages.

Yes.For include `matches_won_this_series` and `num_matches_to_win_series` to game state messages, game server needs to set config below.But almost server does not set it up.
```
mp_teamscore_max "2" # for a BO3 series
mp_teamscore_1 "1"
mp_teamscore_2 "0"
```

>As an ugly workaround I read the scores from a (newly added) teamScores.json file instead that is reread every round-event.

I appreciate your works.It looks good.
But I think we don't need to show BO3/BO5 markers if game state messages is not provide `matches_won_this_series` and `num_matches_to_win_series`.

@bergmannf @LukasW1337 What do yo think about it?

>this is what it would look like: marker example

Very nice design! Nice work!!
